is_iot_sink/mqtt/mqtt_client_base.py

```diff
@@ -0,0 +1,45 @@
+import queue
+import os
+import paho.mqtt.client as mqtt
+from is_iot_sink.logger import LOG
+from is_iot_sink.settings import Settings
+
+class MQTTClientBase:
+    def __init__(self, settings: Settings, name = ""):
+        self._settings = settings
```

Comment:
we should decide on a convention and respect it regarding to using 1 or 2 underscore for private class members

Reply:
In python, double underscore `__` means private variables and single underscore `_` means protected.